doc/book/plugins.md

```diff
@@ -40,35 +40,44 @@ As an example:
 
 ```php
 use Zend\Mvc\Controller\AbstractActionController;
-use Zend\View\Model\JsonModel;
 
 class SomeController extends AbstractActionController
 {
-   protected $acceptCriteria = [
-      'Zend\View\Model\JsonModel' => [
-         'application/json',
-      ],
-      'Zend\View\Model\FeedModel' => [
-         'application/rss+xml',
-      ],
-   ];
-
-   public function apiAction()
-   {
-      $viewModel = $this->acceptableViewModelSelector($this->acceptCriteria);
-
-      // Potentially vary execution based on model returned
-      if ($viewModel instanceof JsonModel) {
-         // ...
-      }
-   }
+    protected $acceptCriteria = [
+        // Make sure ViewModel is the first as a fallback
```

Comment:
Fair point.  I would then just change the wording slightly as it assumes that I want HTML as the fallback.  Maybe "The first model in the list will be used as the default in case of no exact match", or similar ?